src/Pig.cpp

```diff
@@ -109,7 +109,7 @@ bool Pig::skipPartial (const std::string& reference, std::string& result)
   auto pos = 0;
   while (reference[pos] &&
          (*_text)[_cursor + pos] &&
-         reference[pos] == (*_text)[_cursor + pos])
+         reference[pos] == tolower((*_text)[_cursor + pos]))
```

Comment:
I don't think modifying the behaviour of `skipPartial` is the way to go. While this is currently only used for detecting the named dates in `Datetime.cpp`, the name of the method does not suggest that the comparison is based on case-insensitive search. Its counterpart - the `skipLiteral` method - is not case insensitive either.

What I'd suggest is either implementing a new method for this (`skipPartialInsensitive`?), or adding a boolean flag here.

Reply:
Yes that makes sense! I will take a closer look either later today or later this week :+1: